src/baklava.ts

```diff
@@ -1,5 +1,9 @@
-export { default as BlIcon } from './components/icon/bl-icon';
-export { default as BlButton } from './components/button/bl-button';
-export { default as BlInput } from './components/input/bl-input';
-export { default as BlBadge } from './components/badge/bl-badge';
-export { getIconPath, setIconPath } from './utilities/asset-paths';
+export {default as BlIcon} from './components/icon/bl-icon';
+export {default as BlButton} from './components/button/bl-button';
+export {default as BlInput } from './components/input/bl-input';
+export { default as BlBadge} from './components/badge/bl-badge';
+export {default as BlTab} from './components/tab/bl-tab'
+export {default as BlTabGroup} from "./components/tab-group/bl-tab-group";
+export {default as BlTabPanel} from "./components/tab-panel/bl-tab-panel";
+
+export {getIconPath, setIconPath} from './utilities/asset-paths';
```

Comment:
There should be an empty space before `}` and after `{`. (Like as is in `next` branch)